core/src/main/java/tc/oc/pgm/map/MapInfoImpl.java

```diff
@@ -265,6 +266,25 @@ public Component getStyledName(MapNameStyle style) {
         .collect(StreamUtils.toImmutableList());
   }
 
+  private Component parseGamemode(Element root) throws InvalidXMLException {
+    Component gamemode = XMLUtils.parseFormattedText(Node.fromLastChildOrAttr(root, "game"));
+
+    for (Element blitzTitle : XMLUtils.flattenElements(root, "blitz", "title")) {
+      if (this.getProto().isNoOlderThan(MapProtos.REMOVE_BLITZ_TITLE)) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is no longer supported, use <map game=\"...\"> or <game>",
+            blitzTitle);
+      } else if (gamemode != null && blitzTitle != null) {
+        throw new InvalidXMLException(
+            "<title> inside <blitz> is overriden by <map game=\"...\"> or <game>", blitzTitle);
```

Comment:
this exception did not exist in original pgm, so it's a breaking change. It was instead ignored if an actual game element was already defined earlier.

just add an if != null and return the value

Reply:
I was trying to be explicit about the override to avoid confusion about which title is used because whatever I did before didn't result in `<title>` being regarded as an unused node. Should be fine in latest push.